cli/azd/pkg/project/project.go

```diff
@@ -9,10 +9,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/azure/azure-dev/cli/azd/internal"
 	"github.com/azure/azure-dev/cli/azd/internal/telemetry"
 	"github.com/azure/azure-dev/cli/azd/internal/telemetry/fields"
 	"github.com/azure/azure-dev/cli/azd/pkg/ext"
 	"github.com/azure/azure-dev/cli/azd/pkg/osutil"
+	"github.com/blang/semver/v4"
```

Comment:
Is this a new package reference? If so do we need to update the go.mod file?

Reply:
> github.com/blang/semver/v4

It is not, we were already using it different areas of the system (e.g. many of our external tool packages use it to parse version numbers and do comparisons when we want to do things like "check to see we have at least version X of `dotnet`).